pandas/tests/indexes/test_category.py

```diff
@@ -207,19 +207,20 @@ def test_map(self):
         ci = pd.CategoricalIndex(list('ABABC'), categories=list('CBA'),
                                  ordered=True)
         result = ci.map(lambda x: x.lower())
-        exp = pd.Categorical(list('ababc'), categories=list('cba'),
-                             ordered=True)
-        tm.assert_categorical_equal(result, exp)
+        exp = pd.CategoricalIndex(list('ababc'), categories=list('cba'),
```

Comment:
Hi @jreback.  Just wanted to touchbase here since I added this example and then you asked me to remove in another comment below.  Did I add it in the wrong place or did you just decide it was a little overkill?  Thanks.